resources/assets/js/mixins/status.js

```diff
@@ -19,14 +19,16 @@ export default {
     },
     methods: {
         statusUpdate: function (err, data) {
-            if (err && err.status) {
-                this.status.error = 'Error '
-                    + err.status
-                    + ' -  '
-                    + (err.body.message || 'Neem een print screen en neem contact op met de servicedesk.');
-            }
-            else if (err) {
-                this.status.error = err.message;
+            if (err) {
+                if(err.body.error) {
+                    this.status.error = 'Error'
+                        + err.status
+                        + ' - '
+                        + 'Neem een print screen en neem contact op met de servicedesk.';
```

Comment:
@Jelle-S moeten we jouw telefoon nr hier niet zetten als helpdesk? xD

Reply:
![image](https://user-images.githubusercontent.com/1828542/32603896-b9409a4e-c54b-11e7-8026-4add8f38b89e.png)